itests/shellenv.feature

```diff
@@ -0,0 +1,13 @@
+Feature: env variables
+
+Scenario: JBANG_RUNTIME_SHELL available
+When command('jbang env@jbangdev jbang')
+Then match err contains "^JBANG_RUNTIME_SHELL"
+
+Scenario: JBANG_STDIN_NOTTY available
+  When command('jbang env@jbangdev jbang')
+  Then match err contains "^JBANG_STDIN_NOTTY"
+
+Scenario: JBANG_LAUNCH_CMD available
```

Comment:
I actually prefer if we call this `_JBANG_LAUNCH_CMD` but kept with `JBANG_` as we use that above already.

Reason beeing we actually have a few JBANG_* env var used for input - so wanted _JBANG to signal its not input just temporarily available.

Reply:
I said I didn't care about the name, but I don't like starting env var names with non-alpha chars, so I vote for `JBANG_LAUNCH_CMD`.